src/app/signOut/signOut.tpl.html

```diff
@@ -1,13 +1,24 @@
 <div class="screenContent">
-  <div class="redirect-user-after-login" ng-if="$ctrl.showRedirectingLoadingIcon">
-    <div>
-      <loading>
-        <p><translate>Successfully logged out.</translate></p>
-        <p><translate>Redirecting to your previous page...</translate></p>
-      </loading>
-      <button id="closeButtonTop" type="button" ng-click="$ctrl.closeRedirectingLoading()" class="btn btn-block btn-secondary" aria-label="{{'Close' | translate}}">
-        Close <span aria-hidden="true">×</span>
-      </button>
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12 col-xs-12">
+        <div class="panel sign-in-panel">
```

Comment:
Is the `sign-in-panel` class a copy-paste error or just a name that's not completely accurate anymore?